services/ui-src/src/components/cards/EvaluationPlanEntity.tsx

```diff
@@ -0,0 +1,131 @@
+// components
+import { Heading, Text, Grid, GridItem, Flex } from "@chakra-ui/react";
+import { notAnsweredText } from "../../constants";
+// utils
+import { AnyObject, HeadingLevel, ReportType } from "types";
+import { useStore } from "utils";
+
+export const EvaluationPlanEntity = ({
+  formattedEntityData,
+  headingLevel = "h2",
+}: Props) => {
+  const { report } = useStore() ?? {};
```

Comment:
Looks like `const { report } = useStore();` is fine because `report?` is used when needed.